src/jupyter_contrib_nbextensions/nbextensions/toc2/main.js

```diff
@@ -99,48 +112,102 @@ define(["require", "jquery", "base/js/namespace",  'services/config',
   };
   
 
-  var load_ipython_extension = function () {
-    load_css(); //console.log("Loading css")
-    toc_button(); //console.log("Adding toc_button")
-    
-    // read configuration, then call toc
-    cfg = read_config(cfg, 
-      function(){table_of_contents(cfg,st);} // called after config is stable
-      );     
-    
-    // render toc for each markdown cell modification
-    $([IPython.events]).on("rendered.MarkdownCell", 
-      function(){
-        table_of_contents(cfg,st);
-      });
-        console.log("toc2 initialized")
-
-    // add a save as HTML with toc included    
-    addSaveAsWithToc(); 
-
-    // render toc on load 
-    $([IPython.events]).on("notebook_loaded.Notebook", function(){
-        table_of_contents(cfg,st); 
-        console.log("toc2 initialized (via notebook_loaded)")
-})
-
-    // render toc if kernel_ready and add/remove a menu
-    $([IPython.events]).on("kernel_ready.Kernel", function(){
-      console.log("kernel_ready.Kernel")
-      table_of_contents(cfg,st); 
-      console.log("toc2 initialized (via kernel_ready)")
-      // If kernel has been restarted, or changed, check if save_html_with_toc has to be included or removed
-      var IPythonKernel=(IPython.notebook.kernel.name == "python2" || IPython.notebook.kernel.name == "python3")
-        if (!IPythonKernel) {
-            $('#save_html_with_toc').remove()
-        }
-        else{
-          if ($('#save_html_with_toc').length==0) addSaveAsWithToc();
-        }
-    });
+  function create_highlights_css() {
+      var sheet = document.createElement('style')
+      sheet.innerHTML = `#toc-level0 li > a:hover {  display: block; background-color: ${cfg.colors.hoover_highlight} }
```

Comment:
The backtick is not supported in android browser or internet explorer (see https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Template_literals#Browser_compatibility), so it might be a good idea to use a more basic string concatenation operation here, as otherwise it'll presumably be a syntax error, causing the whole nbextension to crash & burn, rather than failing mor gracefully.